### ISSUE TITLE
Blog about r/functionalprogramming

### ISSUE THREAD

**Issue by kinow** [kinow/kinoshita.eti.br#115]
![Screenshot_2019-09-15_14-19-05](https://user-images.githubusercontent.com/304786/65025928-ad23d900-d98b-11e9-86fe-70163566f433.png)

![Screenshot_2019-09-15_14-19-34](https://user-images.githubusercontent.com/304786/65025942-b1e88d00-d98b-11e9-94a6-7d95fb7ac289.png)

![Screenshot_2019-09-15_14-20-01](https://user-images.githubusercontent.com/304786/65025954-b745d780-d98b-11e9-9e0c-0a72eae7048f.png)

**Comment by kinow** [comment]
A community for 6 years.

**Comment by kinow** [comment]
Six years ago I was looking into functional programming, especially around Java, as I was interested in contributing to [Apache Commons Functor](http://commons.apache.org/proper/commons-functor/). At that time Scala was attracting a lot of new users, and Java 8 wasn't available yet.

So I started collecting links. First on one of those post-it apps. But then I wouldn't be able to access it from both home and work. Then started sending e-mails to myself, or keeping a draft e-mail on the server. This way I could access from both home and work, but it wasn't easy to search and organize the links.

Finally, I settled for Reddit. I used it pretty much every day, so I decided to start adding the links to some existing community and simply saving them for later. To my surprise, there was no subreddit for functional programming.

And creating one was extremely simple. That's how six years ago r/functionalprogramming was created.
